src/Store/FileExplorerStore.re

```diff
@@ -6,102 +6,154 @@
 open Oni_Core;
 open Oni_Model;
 
-let start = () => {
-  let (stream, _) = Isolinear.Stream.create();
-
-  let getExplorerFilesEffect = (cwd, languageInfo, iconTheme, ignored) => {
-    Isolinear.Effect.createWithDispatch(~name="explorer.open", dispatch => {
+module Effects = {
```

Comment:
Nice 👍

Reply:
I don't think we can put most effects in `Effects` submodules in the stores though, because it'll cause dependency cycles. But then I also think very few updaters should be in "stores". They should instead be associated more with UI components since that's what the "actions"/"events" will refer to. But it still makes sense to have a "VimStore" for example.